Vulnerabilities found

[Govulncheck](https://pkg.go.dev/golang.org/x/vuln/cmd/govulncheck) has found some issues in the latest GitHub Action run: <https://github.com/kubecolor/kubecolor/actions/runs/10769210560>.

Full output:

````console
$ govulncheck ./...

````

Nawe the output capture failed.

The output it should have shown:

```console
$ govulncheck ./...
=== Symbol Results ===

Vulnerability #1: GO-2024-3107
    Stack exhaustion in Parse in go/build/constraint
  More info: https://pkg.go.dev/vuln/GO-2024-3107
  Standard library
    Found in: go/build/constraint@go1.23
    Fixed in: go/build/constraint@go1.23.1
    Example traces found:
      #1: internal/cmd/configdoc/parse.go:32:28: configdoc.ParseCategories calls parser.ParseFile, which eventually calls constraint.Parse

Vulnerability #2: GO-2024-3105
    Stack exhaustion in all Parse functions in go/parser
  More info: https://pkg.go.dev/vuln/GO-2024-3105
  Standard library
    Found in: go/parser@go1.23
    Fixed in: go/parser@go1.23.1
    Example traces found:
      #1: internal/cmd/configschema/main.go:45:17: configschema.main calls jsonschema.Reflector.AddGoComments, which eventually calls parser.ParseDir
      #2: internal/cmd/configdoc/parse.go:32:28: configdoc.ParseCategories calls parser.ParseFile

Your code is affected by 2 vulnerabilities from the Go standard library.
This scan also found 0 vulnerabilities in packages you import and 1
vulnerability in modules you require, but your code doesn't appear to call these
vulnerabilities.
Use '-show verbose' for more details.
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM docker.io/library/golang:1.22.5 AS build
+FROM docker.io/library/golang:1.22.7 AS build
 
 WORKDIR /go/src/kubecolor
 COPY go.mod go.sum .

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/kubecolor/kubecolor
 
-go 1.22.5
+go 1.22.7
 
 require (
 	github.com/MakeNowJust/heredoc v1.0.0

